### ISSUE TITLE
Neural networks from scratch

### ISSUE THREAD

**Issue by thylakoids** [thylakoids/thylakoids.github.io#4]
1. Why using cross-entropy loss?
